.readthedocs.yaml

```diff
@@ -1,5 +1,10 @@
 version: 2
-sphinx:
-  configuration: docs/source/conf.py
+
+build:
+  os: ubuntu-22.04
```

Comment:
just needed to backfill this to get a preview build :heart: 
- build https://readthedocs.org/projects/lumino/builds/22062754/
- preview https://lumino--633.org.readthedocs.build/en/633/examples/dockpanel/index.html